### DISCUSSION TITLE
Is there a Shi no numa map?

### DISCUSSION THREAD

**Discussion by M3M3RIST** [nzp-team/nzportable#879] [General]
i wanted to play the original zombies maps, like kino der toten, verruckt, der riese, shi no numa etc. i already have kino der toten and der riese, but is there a shi no numa and verruckt map? please let me know.

**Comment by radiohead-sucks** [comment]
> is there a shi no numa and verruckt map?

doesn't look like it, but there's work being done on verruckt (there's even a [teaser](https://www.youtube.com/watch?v=c6W4jW0kXoI) for it)

**Reply by M3M3RIST** [reply]
Verruckt is good, but i really want to play shi no numa, thanks for the info though.
